### DISCUSSION TITLE
Is there a way to command line execute a saved out workflow graph json?

### DISCUSSION THREAD

**Discussion by Fredrum** [comfyanonymous/ComfyUI#3132] [Q&A]
Hello!
I love working with Comfy as it makes it relatively easy chaining different workflows together and customize things to get closer to the images I'm looking for.

But I'd like to be able to execute these workflows without using the UI and from inside a program like python or c++ for example.
It seems like it should be possible since all the info about the setup is stored in the json file. 

Does anyone know if and how I could do this?

Cheers!

**Comment by UrzasLegacy** [comment]
https://github.com/comfyanonymous/ComfyUI/tree/master/script_examples

This should get ya pointed in the right direction

**Reply by Fredrum** [reply]
That's awesome thank you very much!
I will try it out!

**Reply by andrewtvuong** [reply]
> https://github.com/comfyanonymous/ComfyUI/tree/master/script_examples
> 
> This should get ya pointed in the right direction

Thanks dude was looking for this too!
